scanomatic/data_processing/phenotyper.py

```diff
@@ -322,73 +328,62 @@ def LoadFromState(cls, directory_path: str) -> "Phenotyper":
         smooth_growth_data = unpickle_with_unpickler(
             np.load,
             os.path.join(directory_path, _p.phenotypes_input_smooth),
+            allow_pickle=True,
         )
 
         try:
-            extraction_params = unpickle_with_unpickler(
-                np.load,
-                os.path.join(directory_path, _p.phenotypes_extraction_params),
-            )
+            extraction_params = jsonizer.load(os.path.join(
```

Comment:
What type is this? It should be the dataclass it used to be, but it looks like it is now a dict and if it is, I think it will be quite problematic. If it is a dict I would argue there should be a new jsonizer serializer/deserializer for dataclasses just like enums and Model:s

Reply:
No, previously it was just an array, not a dataclass.

Reply:
When it is created in the app before serialization it is a dataclass so it should be that when loaded

Reply:
Yes, this whole method is about creating that dataclass from the json.

Reply:
no it's about loading a phenotyper state, but the settings should just be `settings: PhenotyperSettings = jsonizer.load(the_path)`

Reply:
most, if not all, from this line to line 395 should be removed (and should not be needed in the deserializer in jsonizer

Reply:
It that would work, that would be lovely. Bearing that in mind though, you did tell me to take short cuts, and I think this, besides the fact that it works, is already an improvement on how it used to work. Please make an executive call on what you think is appropriate.

Reply:
I think jsonizer should load it

Reply:
fixed